src/Intercom/Data/SocialProfile.cs

```diff
@@ -11,7 +11,7 @@ namespace Intercom.Data
 {
 	public class SocialProfile : Model
 	{
-		public string type { get; set; }
+		public override string type { get; set; }
```

Comment:
Nice one, we  can actually remove `public string type { get; set; } ` all together, since it is in the parent 👍